Tests/VisualRecognitionV3Tests/VisualRecognitionTests.swift

```diff
@@ -1126,7 +1132,9 @@ class VisualRecognitionTests: XCTestCase {
                 } else {
                     // verify the image's custom classifier
                     XCTAssertEqual(classifier.classifierID, self.classifierID!)
-//                    XCTAssertEqual(classifier.name, self.classifierName)
+                    if classifier.classifierID != classifier.name {
```

Comment:
Ahh right, I remember you telling me about that now. Hmm. Do you think they would eventually change it back to the "correct" response, and actually send back the proper classifier name? Do they have plans to?

I ask because I wonder if we should keep the test as it was. I feel like it might be better to keep a "correct" test, rather than change our tests based on erroneous data from the service, just to make sure our tests pass.